src/Node/Node.php

```diff
@@ -30,6 +30,10 @@ class Node implements \Countable, \IteratorAggregate
     protected $tag;
 
     private $sourceContext;
+    /** @var NameDeprecation[] */
```

Comment:
I suggest using `array<string, NameDeprecation>` to document the type of keys

Reply:
Done